deps/Versions.make

```diff
@@ -1,18 +1,32 @@
 LLVM_VER = 6.0.1
-LLVM_BB_REL = 3
-PCRE_VER = 10.30
+LLVM_BB_REL = 5+nowasm
+PCRE_VER = 10.32
+PCRE_BB_REL = 0
 DSFMT_VER = 2.2.3
+OPENBLAS_VER = 0.3.5
+OPENBLAS_BB_REL = 0
 LAPACK_VER = 3.5.0
 SUITESPARSE_VER = 5.4.0
 SUITESPARSE_BB_REL = 1
 OPENLIBM_VER = 0.6.0
+OPENLIBM_BB_REL = 0
 UNWIND_VER = 1.3.1
 OSXUNWIND_VER = 0.0.5
 GMP_VER = 6.1.2
+GMP_BB_REL = 0
 MPFR_VER = 4.0.2
+MPFR_BB_REL = 0
 PATCHELF_VER = 0.9
 MBEDTLS_VER = 2.16.0
-CURL_VER = 7.56.0
+MBEDTLS_BB_REL = v0.17.0
```

Comment:
Just a thumbs up here on the right version here for mbedtls 👍 . That's about all I have to contribute in terms of review!